.github/workflows/ci.yml

```diff
@@ -18,26 +18,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - elixir: 1.10.x
-            otp: 21.3.8.21
-          - elixir: 1.10.x
-            otp: 22.3.4.16
-          - elixir: 1.10.x
-            otp: 23.2.7
-            warnings_as_errors: true
-            static_analysis: false
-          - elixir: 1.11.x
-            otp: 21.3.8.21
-          - elixir: 1.11.x
-            otp: 22.3.4.16
-          - elixir: 1.11.x
-            otp: 23.2.7
-            warnings_as_errors: true
-            static_analysis: false
-          - elixir: 1.12.x
```

Comment:
I dont think we want to remove opt 22 and 23 from the 1.12 matrix.  Phoenix currently officially supports (as of Phx 1.6) Elixir 1.12+ and OTP 22+